examples/collaborative-text-editor/Procfile.dev

```diff
@@ -1,4 +1,4 @@
 web: unset PORT && bin/rails server
 js: yarn build:js --watch
 css: bin/rails tailwindcss:watch
-redis: redis-server
+redis: docker run --rm --name redis -p 6379:6379 redis:7-alpine
```

Comment:
Mount the Redis data directory as volume in `tmp/redis`.

```suggestion
redis: docker run --rm --name redis -p 6379:6379 -v "$(pwd)"/tmp/redis:/data \ redis:7-alpine
```

Reply:
`tmp/redis` does not exist for me in this tree; we will have to `mkdir -p` this first I think?